client/src/store/comments/actions.ts

```diff
@@ -50,15 +50,18 @@ export const setDefaultCommentToChange = (postId: string): { type: string, paylo
 });
 
 export const onChangeComment = (postId: string, comment: string):
-{ type: string, payload: { postId: string, comment: string } } => ({
-    type: ON_CHANGE_COMMENT,
-    payload: { postId, comment },
-});
+    { type: string, payload: { postId: string, comment: string } } => ({
+        type: ON_CHANGE_COMMENT,
+        payload: {postId, comment},
+    });
 
-export const addNewComment = (newComment: IComments): { type: string, payload: any } => ({
-    type: ADD_COMMENT,
-    payload: newComment,
-});
+export const addNewComment = (newComment: any): { type: string, payload: any } => {
```

Comment:
why sis u change all types to any?

Reply:
I did deep investigate in our code and usually used "any" because it didn't take much time
And at the end forgot to change it in some places

Reply:
fixed